src/app/[locale]/(profile)/(components)/Filters/ProfileFilters.tsx

```diff
@@ -29,6 +30,7 @@ export const ProfileFilters = ({
   technologies,
   countries,
 }: ProfileFiltersProps) => {
+  const t = useTranslations('DropdownFilter')
```

Comment:
I'm not sure if the enum will work, the documentation strictly mentions the lang.json files in the messages folder. The names are unique, I built the json files so that they do not duplicate each other, we have the names of the components and the appropriate translations under them.